core/src/main/kotlin/builder/kord/KordProxyBuilder.kt

```diff
@@ -0,0 +1,44 @@
+package dev.kord.core.builder.kord
+
+import dev.kord.cache.api.DataCache
+import dev.kord.common.annotation.KordExperimental
+import dev.kord.common.entity.Snowflake
+import dev.kord.core.ClientResources
+import dev.kord.core.Kord
+import dev.kord.core.gateway.DefaultMasterGateway
+import dev.kord.core.supplier.EntitySupplyStrategy
+import dev.kord.gateway.Gateway
+import dev.kord.gateway.builder.Shards
+import dev.kord.rest.service.RestClient
+import kotlinx.coroutines.flow.MutableSharedFlow
+
+/**
+ * The rest only Kord builder. You probably want to invoke the [DSL builder][Kord.restOnly] instead.
+ */
+@KordExperimental
+public class KordProxyBuilder: RestOnlyBuilder() {
+
+    public override fun build(): Kord {
```

Comment:
there is still a lot of duplicated logic, can we put more of it in the common base class?

Reply:
You are right we can add duplicate code into the abstract class as well